flow/designs/intel22/ibex/config.mk

```diff
@@ -0,0 +1,34 @@
+$(info [INFO-FLOW] AES Design)
+DESIGN_DIR                   := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+DESIGN_PDK_HOME              := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+
+export DESIGN_NICKNAME = ibex
+export DESIGN_NAME = ibex_core
+export PLATFORM    = intel22
+
+export VERILOG_FILES = $(sort $(wildcard $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/*.v))
+export SDC_FILE      = $(DESIGN_DIR)/constraint.sdc
+
+export CORE_UTILIZATION = 30
+export CORE_ASPECT_RATIO = 1
+export CORE_MARGIN = 2
+
+export PLACE_DENSITY = uniform
+
+export CORNER                ?= BC
+
+export LIB_FILES             += $($(CORNER)_LIB_FILES)
+export LIB_DIRS              += $($(CORNER)_LIB_DIRS)
+export DB_FILES              += $($(CORNER)_DB_FILES)
+export DB_DIRS               += $($(CORNER)_DB_DIRS)
+export WRAP_LIBS             += $(WRAP_$(CORNER)_LIBS)
+export WRAP_LEFS             += $(WRAP_$(CORNER)_LEFS)
+export TEMPERATURE            = $($(CORNER)_TEMPERATURE)
+
+export ABC_CLOCK_PERIOD_IN_PS = 8000
+
+export DESIGN_POWER           = VDD
+export DESIGN_GROUND          = VSS
```

Comment:
What are these for?

Reply:
Removed it was there as legacy -- I dont see it beiong used